### PR TITLE
feat/KuCoin connector part 1: DataSource & OrderBookTracker

### PR DESCRIPTION
**Pull request by vic-en**
KuCoin DataSource and OrderBookTracker & Tests

### CODE REVIEW

File: conf/__init__.py [hunk 1]
```diff
@@ -1,99 +0,0 @@
-#!/usr/bin/env python
-
-import os
```
Comment (CrimsonJacket) [inline conf/__init__.py]: There is no need to delete this file

**Comment by CrimsonJacket** [comment]
@vic-en It seems that you have not refactored `symbol`->`trading_pair`. I have chose to approve this PR. But in subsequent PRs, you are to ensure that this refactor is done right.

**Comment by yzhang1994** [comment]
test this please

**Comment by CrimsonJacket** [comment]
ok to test

**Pull request merged**